WordPressEditor/src/main/java/org/wordpress/android/editor/EditorFragment.java

```diff
@@ -740,13 +738,13 @@ public void run() {
                 if (URLUtil.isNetworkUrl(mediaUrl)) {
                     String mediaId = mediaFile.getMediaId();
                     mWebView.execJavaScriptFromString("ZSSEditor.insertImage('" + mediaUrl + "', '" + mediaId + "');");
-                    AnalyticsTracker.track(Stat.EDITOR_ADDED_PHOTO_VIA_WP_MEDIA_LIBRARY);
+                    mEditorFragmentListener.onTrackableEvent(TrackableEvent.NETWORK_MEDIA_ADDED);
```

Comment:
Renamed Media/Image events in 69aaa8a

As per our chat, I'll probably kill `NETWORK_IMAGE_ADDED`, `LOCAL_IMAGE_ADDED`, `UPLOAD_IMAGE_RETRIED` and `UPLOAD_IMAGE_FAILED` and track them in the activity.